Src/WitsmlExplorer.Frontend/components/Modals/SettingsModal.tsx

```diff
@@ -59,6 +66,27 @@ const SettingsModal = (): React.ReactElement => {
     dispatchOperation({ type: OperationType.SetTimeZone, payload: selectedTimeZone });
   };
 
+  const onChangeDecimals = (event: any) => {
+    const inputDecimals: any = parseInt(event.target.value, 10);
+    if (!isNaN(inputDecimals) && inputDecimals >= 0 && inputDecimals <= 10) {
+      setDecimalError(false);
+      localStorage.setItem(STORAGE_DECIMAL_KEY, inputDecimals);
+      dispatchOperation({ type: OperationType.SetDecimal, payload: inputDecimals });
+    } else {
+      setDecimalError(true);
+    }
+  };
+
+  const onChangecheck = (event: React.ChangeEvent<HTMLInputElement>) => {
+    const selectedValue = event.target.value;
+    if (selectedValue === DecimalPreference.Raw) {
+      const selectedDecimals = DecimalPreference.Raw;
+      localStorage.setItem(STORAGE_DECIMAL_KEY, selectedDecimals);
+      dispatchOperation({ type: OperationType.SetDecimal, payload: selectedDecimals as DecimalPreference });
```

Comment:
remove `selectedDecimals` and use `DecimalPreference.Raw` directly in `dispatchOperation`